doc/design/ant_xgboost.md

```diff
@@ -71,7 +71,7 @@ USING sqlflow_models.xgboost_model_table;
 As `codegen.go` generating TensorFlow code from sqlflow AST,
 we will add `codegen_xgboost.go` which translate sqlflow AST into a python launcher program of xgboost. 
 
-Since xgblauncher provide `DataSource` and `ModelSource`, abstraction of custom I/O pipeline, by which we can reuse data/model pipeline of `sqlflow_submitter`.
+Since xgblauncher provide `DataSource` and `ModelSource`, abstraction of custom I/O pipeline, by which we can reuse data/model pipeline of `sqlflow_runtime`.
```

Comment:
If the Ant-XGBoost codegen is obsolete, how about we remove this document in a future PR?

Reply:
Sure, would remove it in the next PR.